cl/search/feeds.py

```diff
@@ -97,6 +97,38 @@ def item_title(self, item):
     def item_categories(self, item):
         return [item['status'], ]
 
+    def item_enclosure_url(self, item):
+        try:
+            path = item['local_path']
+            if not path.startswith('/'):
+                return '/%s' % (path,)
```

Comment:
This is the Django Feed object will render it as path off the root of the site vs. a relative path to the feed URL. Without it, you end up with something like `https://www.courtlistener.com/feeds/court/scotus/pdf/2015/12/30/filename.pdf`
instead of
`https://www.courtlistener.com/pdf/2015/12/30/filename.pdf`